### ISSUE TITLE
TypeError: translation[i] is undefined (French)

### ISSUE THREAD

**Issue by keichitokuna** [h5p/h5p-course-presentation#244]
Hello,

When i try to add an interactive video inside Course Presentation, i am stuck at "Loading, please wait..." (written in French) with this error in console :
![Capture d’écran 2022-11-16 à 10 34 40](https://user-images.githubusercontent.com/13786607/202143882-1afd9b0e-f686-40bc-8202-6dbcc9d24ae5.png)

If i use moodle in English, everything is working as expected.

Here is a video : https://cirrus.universite-paris-saclay.fr/s/kDGemEe6kX4369H

I use Course Prensentation 1.24.1 up to date.

Thanks,

K.

**Comment by otacke** [comment]
@keichitokuna https://h5p.org/node/1295740

**Comment by otacke** [comment]
@keichitokuna No, as mentioned in the forum thread, you'll need version 1.24.4 of Interactive Video.

**Comment by keichitokuna** [comment]
Indeed, the problem is now solved. Thanks !
